src/Posting/QR.coffee

```diff
@@ -166,6 +166,16 @@ QR =
     else
       QR.unhide()
 
+  toggleAddCooldown: ->
+    if @checked
+      QR.cooldown.addCooldown = true
+      if QR.cooldown.seconds
+        QR.cooldown.seconds += parseInt(Conf['additionalCooldown'])
+    else
+      QR.cooldown.addCooldown = false
+      if QR.cooldown.seconds
+        QR.cooldown.seconds -= parseInt(Conf['additionalCooldown'])
```

Comment:
I'm not sure if we need to change `QR.cooldown.seconds` here at all, since it will be updated by the countdown anyway.  But if we do, we should guard against it going negative.